README.md

```diff
@@ -1,15 +1,35 @@
-== What is mydumper? Why? ==
+# What is mydumper? Why?
 
 * Parallelism (hence, speed) and performance (avoids expensive character set conversion routines, efficient code overall)
 * Easier to manage output (separate files for tables, dump metadata, etc, easy to view/parse data)
 * Consistency - maintains snapshot across all threads, provides accurate master and slave log positions, etc
 * Manageability - supports PCRE for specifying database and tables inclusions and exclusions
 
-== How to build it? ==
+## How to install mydumper/myloader?
+
+First get the correct url from the [releases section](https://github.com/maxbube/mydumper/releases) then:
```

Comment:
could we change these link?

Reply:
I've updated the `README` just now to make it more clear what this fork does.  The manual links will not work until https://github.com/pingcap/docs/pull/644 is merged.